tests/testthat/test-project-elements.R

```diff
@@ -186,20 +184,23 @@ test_that('The add_funding function adds the funding elements', {
 })
 
 test_that('The default funding options are adding the correct default funding information', {
-  expect_equal(add_funding(CVPIA_default_funder = "CDWR"),
+  expect_equal(add_funding(funder_name = "CDWR", 
```

Comment:
I was thinking you would access funder_name like this for the tests: `EDIutils::funders$CDWR`